search-books/src/main/java/nix/education/service/suggestions/SolrBookAuthorSuggestionService.java

```diff
@@ -0,0 +1,59 @@
+package nix.education.service.suggestions;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import lombok.SneakyThrows;
+import nix.education.util.SymbolsSolrQuery;
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.response.QueryResponse;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SolrBookAuthorSuggestionService {
```

Comment:
It's a good practice to implement some interface when declaring a service

Reply:
Ok, I'll create interface for this class